tests/models/test_pyro.py

```diff
@@ -169,7 +230,7 @@ def test_pyro_bayesian_regression_jit():
     trainer.fit(plan, train_dl)
 
     # 100 features, 1 for sigma, 1 for bias
-    assert list(model.guide.parameters())[0].shape[0] == 102
+    # assert list(model.guide.parameters())[0].shape[0] == 102
```

Comment:
why comment this out?

Reply:
@adamgayoso AutoNormal guide does not have a single parameter - it has one locs and one scales parameter per variable (and I think it has to be used when the model has plates). What are you trying to test with this?

Reply:
right there should be 102 variables (for each of loc and scale), 100 for the weight vector, 1 for the sigma from the sample statement and one for the liner bias

Reply:
This test fails with AutoNormal guide probably because these are 3 separate parameters

Reply:
I think what this test was doing was to ensure the guide was properly initialized, otherwise you don't get latent var params for each latent var

Reply:
Not sure I understand what you mean. Do you mean it tests that the number and shape of all latent variable corresponds to the number and shape of guide param? If yes, the code I added in replacement tests that for weight vector: https://github.com/YosefLab/scvi-tools/pull/1059/files#diff-663073fe4ef3cfa8f3fad68a502f760ee33dbb94c3c803c50635f0d63f67ee30R278-R280